CourseSketchProjects/CourseSketchDatabaseServer/src/main/java/database/institution/mongo/CourseManager.java

```diff
@@ -351,10 +352,10 @@ static boolean mongoInsertAssignmentIntoCourse(final DB dbs, final String course
      */
     static boolean mongoInsertLectureIntoCourse(final DB dbs, final String courseId, final String lectureId)
             throws AuthenticationException, DatabaseAccessException {
-        final DBRef myDbRef = new DBRef(dbs, COURSE_COLLECTION, convertStringToObjectId(courseId));
-        final DBObject cursor = myDbRef.fetch();
+        final DBCollection collection = dbs.getCollection(getCollectionFromType(School.ItemType.COURSE));
+        final DBObject cursor = collection.findOne(convertStringToObjectId(courseId));
         DBObject updateObj = null;
-        final DBCollection courses = dbs.getCollection(COURSE_COLLECTION);
+        final DBCollection courses = dbs.getCollection(getCollectionFromType(School.ItemType.COURSE));
```

Comment:
when is this object used and why is it a copy of collection? why not use collection?